Borrower.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+sudo su
+yum install git -y
+yum install java-1.8.0-openjdk -y
+yum install maven -y
+sudo yum update -y
```

Comment:
Work on having these statements inputted by another simple bash script for code re-usability and just being preloaded at bashrc or another bash profile